ddtrace/bootstrap/preload.py

```diff
@@ -100,6 +100,23 @@ def _otel_logs():
     LLMObs.enable(_auto=True)
 
 
+# Enable logging on native components
+if config._native_logging:
```

Comment:
should we add the logging configuration here?

https://github.com/DataDog/dd-trace-py/blob/ae3ff7b58457e78082aaa7ab2c1b480a2d6e25cf/ddtrace/_logger.py


This bootstrap preload is only loaded when `import ddtrace.auto` or `ddtrace-run` are used. Moving to `ddtrace._logger` could help ensure any entrypoint paths can also get native logging configured.

Also, the helpers in `ddtrace._logger` are what we call when adding a file log handler for tracer flares, so could be a good place to also have native file handler configuration located.


One thing of note, for tracer flares we add an additional file handler, so it is possible the customer gives us a file path to send our logs to, and then when a tracer flare is being collected we also send logs to the tracer flare location as well.